app/lib/metric_util.rb

```diff
@@ -150,5 +150,10 @@ def https_post(uri, data)
         end
       end
     end
+
+    def a_test?(request = nil)
```

Comment:
nit: maybe `is_test?` or `is_a_test?` for a boolean function? similar to `is_a?` https://apidock.com/ruby/Object/is_a%3F

Reply:
I originally wrote `is_a_test?` but rubocop complained.